src/components/ImagePreloader.tsx

```diff
@@ -3,15 +3,12 @@ import odFullLogoLight from '../assets/od-full-logo-light.svg'
 import darkArrow from '../assets/dark-arrow.svg'
 import walletConnectIcon from '../assets/connectors/walletConnectIcon.svg'
 import coinbaseWalletIcon from '../assets/connectors/coinbaseWalletIcon.svg'
-import cookie from '../assets/cookie.svg'
```

Comment:
I noticed that we might not use some of the pictures here, I removed them as well (I checked if there were used somewhere else in the code)